Hardcode term 13 NPM start dates

ondenman opened the pull request:
The start dates of NPMs are removed after the takes their seat.
    That is, we once had these start dates but now we haven't.
    Since the official source no longer publishes these dates,
    we're hardcoding them into the scraper.

Closes https://github.com/everypolitician/everypolitician-data/issues/14081

ondenman committed:
Hardcode term 13 NPM start dates

tmtmtmtm commented on scraper.rb:
```diff
@@ -64,7 +64,30 @@ def scrape_term(url)
     data[:end_date] = date_from(data[:notes]) if data[:notes].to_s.include? 'Resigned on '
+    data[:end_date] ||= term[:end_date]
     data[:start_date] = date_from(data[:notes]) if data[:notes].to_s.include? 'NMP term effective '
+    data[:start_date] ||= term[:start_date]
```
I'm not sure why you're adding this (or the equivalent at line 67). It doesn't seem relevant to this change, isn't really necessary, and introduces the potential for a very nasty and subtle bug upstream much later if we discover that the start or end date for a term was actually incorrect.

tmtmtmtm commented on scraper.rb:
```diff
@@ -64,7 +64,30 @@ def scrape_term(url)
     data[:start_date] = date_from(data[:notes]) if data[:notes].to_s.include? 'NMP term effective '
+    data[:start_date] ||= term[:start_date]
+
+    # The start dates of NPMs are removed after the takes their seat.
```
"after the takes" seems to be missing a word.

tmtmtmtm committed:
don't clobber the start/end dates

There's no need to set these, as they're already implied from the term
dates anyway.

tmtmtmtm committed:
simplify list of term_13_nmps

tmtmtmtm committed:
Set the Party for NMPs

Explicitly noted that they're an NMP if no Party is issued.

This should really be set differently from this, but it's consistent
with what we're already doing, so fixing that can be done later.

tmtmtmtm committed:
Comment out open-uri/cached

scraped_page_archive/open-uri doesn't play well with open-uri/cached

tmtmtmtm commented:
@ondenman I fixed this up a little bit so we could have it go live this week. I also added a commit to explicitly set the party for NMPs, as the site only seems to provide it for _some_ of them.
